src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+export interface IBasePickerProps extends React.Props<any> {
+  /**
+   *
+   */
+  onRenderItem?: (item: IPickerItemProps) => JSX.Element;
+  /**
+   *
+   */
+  onRenderSuggestion?: (props: any) => JSX.Element;
+  /**
+   *
+   */
+  onResolveSuggestions: (filter: string) => any[];
+  /**
+   *
+   */
+  onSuggestionClick?: (ev?: React.MouseEvent, item?: any) => void;
+  /**
+   *
+   */
+  startingItems?: any[];
+  /**
+   *
+   */
+  onChange?: (items: any[]) => void;
+  /**
+   *
+   */
+  getTextFromItem?: (T) => string;
+  /**
+   *
+   */
+  suggestionsHeaderText?: string;
+  /**
+   *
+   */
+  onGetMoreResults?: (filter: string) => any[];
+  /**
+   *
+   */
+  className?: string;
+  /**
+   *
+   */
+  suggestionsClassName?: string;
+  /**
+   *
+   */
+  searchForMoreText?: string;
```

Comment:
i don't know why this is here and not in something like a suggestions component. The actual content that's rendered in suggestions should be irrelevant to the picker.

I think this really needs some scrubbing.